### DISCUSSION TITLE
11/28 조지연 블로그 2회차

### DISCUSSION THREAD

**Discussion by oratio100** [Orm-camp/blog-feedback#41] [동글동글 3기 피드백]
[애니메이션 루프](https://velog.io/@nemonemo/three.js-%EC%95%A0%EB%8B%88%EB%A9%94%EC%9D%B4%EC%85%98-%EB%A3%A8%ED%94%84-%ED%95%A8%EC%88%98)
그냥 정보 정리해둔 글이라 별 내용은 없습니다;;;;

**Comment by Orm-camp** [comment]
실습하시면서 그냥 넘어갈 수도 있는 부분이지만, 두 함수의 차이를 찾아보고 정리하셨다는 것이 매우 인상 깊었습니다!
'실행'에만 포커스를 두는 사람이 아니라 '구조'와 '쓰임'에도 관심을 가지고 있는 사람이다 라는 느낌을 크게 받았어요.
이런 포인트를 놓치지 않고 학습하고, 기록하고, 나타내는 게 정말 중요하다고 생각합니다!

**Comment by mh0223** [comment]
애니메이션 루프 함수 중 requestAnimationFrame()도 다뤄주셔서 좋아요! 맨날 쓰던 것만 써서 다른 건 뭔가 .. 따로 찾아보기 망설여졌는데 지연님 글 읽고 쉽게 이해하고 적용도 해볼 수 있을 것 같습니다 ! 애니메이션 중단도 추가로 설명해주셔서 도움이 더 되는 것 같아요!
그리고 저번에 글 소제목이 조금 진했으면 좋겠다고 피드백 드렸는데 그 부분도 뭔가 반영을 하신 것 같아서 좋았습니다

**Comment by kimjigyun** [comment]
설명 , 특징 , 예문 이렇게 나눠서 정리하니까 보기 좋아요 !! 제목 이미지 넣으신거 너무 예뻐욥

**Comment by baaanjy** [comment]
요점 정리가 잘 되어있어서 쉽게 읽히고 이해가 잘되었어요

**Comment by Devkdy22** [comment]
requestAnimationFrame() 과  setAnimationLoop()의 차이점,중단하는 방법에 대해서까지 설명해주셔서 새로운 점들을 알게되었습니다! ☺️👍🏻

**Comment by naseulgee** [comment]
예시 코드와 함께 설명이 되어 있어서 이해가기 좋은 것 같아요!👍

**Comment by hansol-woff** [comment]
항목 별로 잘 나누셔서 보기에 편했습니다! 글의 서론이나 결론으로 아예 두 함수의 차이점을 한 줄로 요약해서 작성하는 것도 좋을 것 같습니다.
